packages/utilities/psammead-assets/src/svgs.stories.jsx

```diff
@@ -61,7 +61,7 @@ getSVG.propTypes = {
   height: number,
 };
 
-const stories = storiesOf('Utilities|SVGs/Brand Svgs', module).addDecorator(
+const stories = storiesOf('UTILITIES/SVGS/Brand Svgs', module).addDecorator(
```

Comment:
Just a minor thing - any reason why Utilities is now all CAPS?

Reply:
Ah must have been because I was REALLY REALLY ANGRY when I wrote it

No absolutely not, I'll revert. Storybook automatically capitalises component folders, I might have copied and pasted from there 🤷‍♂️